source/MaterialXContrib/MaterialXMaya/AEtemplate/AEMaterialXSurfaceTemplate.mel

```diff
@@ -1,27 +1,65 @@
-global proc MaterialXSurface_extraCtrlsBuild(string $nodeName)
```

Comment:
"Extra controls" has a special meaning in AE templates - as in "editorTemplate -addExtraControls".